src/index.d.ts

```diff
@@ -46,6 +46,7 @@ export interface SunburstChartGenericInstance<ChainableInstance> {
   label(textAccessor: NodeAccessor<string>): ChainableInstance;
   size(): NodeAccessor<string>;
   size(sizeAccessor: NodeAccessor<string>): ChainableInstance;
+  transitionDuration(duration: number): ChainableInstance;
```

Comment:
This method is missing the getter type entry:
```js
transitionDuration(): number;
```

Can we also place it further down towards the bottom of the block for consistency with the docs?